common/io.cpp

```diff
@@ -27,34 +27,24 @@ Error Copy(Writer &dst, Reader &src, vector<uint8_t> &buffer) {
 	size_t orig_size = buffer.size();
```

Comment:
`orig_size` is no longer needed, right?

Reply:
Indeed. Why didn't it warn about this..?

Reply:
Does it ever? :thinking: 

I think it's only unhappy about unused functions

Reply:
Time for `-Wextra`?

Reply:
It should have caught it anyway:

> ```
>  -Wunused-but-set-variable
>            Warn whenever a local variable is assigned to, but otherwise unused
>            (aside from its declaration).  This warning is enabled by -Wall.
> ```